### PR TITLE
Add VSCode debugger launch configuration

### PR DESCRIPTION
**Pull request by chosak**
Allows for running the Django dev server in the VSCode debugger. For documentation on this file see:

https://code.visualstudio.com/docs/editor/debugging#_launch-configurations

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": "0.2.0",
```
Comment (anselmbradford) [inline .vscode/launch.json:2]: How does the versioning work? Is this value arbitrary?

Comment (chosak) [reply .vscode/launch.json:2]: I'm not sure tbh, I think it's internal versioning for VSCode launch schema. See in their source code here: https://github.com/microsoft/vscode/blob/4f0c8af89496497d57982f748901c94e75436f43/src/vs/workbench/contrib/debug/common/debugSchemas.ts#L197.

**Review by anselmbradford — APPROVED** [review]
This LGTM, but you may want a bewd to verify. So if we had another debugger, we'd just add a second entry with a different `type`, right?

**Comment by chosak** [comment]
@anselmbradford correct; you can even do multi-target debugging if you wanted to set up a frontend debugger side-by-side with the Django one: https://code.visualstudio.com/docs/editor/debugging#_multitarget-debugging

**Pull request merged**